requirements.txt

```diff
@@ -1,7 +1,5 @@
 aiohttp
 escapism
-etcd3
```

Comment:
Do you want to add these (etcd3 and python-consul2) to `dev-requirements.txt` instead, to fix the tests?

Reply:
That's not a bad idea at all! This *is* a breaking change tho, so we should announce widely. 

I also want to test that the toml provider works without these libraries. How do you think we can do that?

Reply:
I didn't notice any toml proxy test failing in https://github.com/jupyterhub/traefik-proxy/pull/127/checks?check_run_id=2211574725 :heart: Only the consul and etcd ones.

Reply:
Right. But in the future, if we accidentally introduce a change that automatically imports etcd3 or python-consul2, the tests in this repo won't catch it! But for an end user who doesn't have these packages installed but only wants to use the TOML proxy, it will fail. So I want to find a way to test that the toml proxy works even without these packages installed.